app/javascript/components/batch_update/helpers/elasticsearch.js

```diff
@@ -163,11 +163,21 @@ const acquireableOrOfferableMatcher = acquireableOrOfferableFilter => {
   switch (acquireableOrOfferableFilter) {
     case 'SHOW_ACQUIREABLE_OR_OFFERABLE':
       return {
-        or: [{ term: { offerable: true } }, { term: { acquireable: true } }],
+        bool: {
+          should: [
```

Comment:
Ah interesting. `Should` feels to me like it denotes some kind of fuzzy search rather than a simple `or` — I'll get used to it I guess!